### PR TITLE
Reorder polyfills, add popover demo link

### PR DESCRIPTION
**Pull request by jgerigmeyer**

### CODE REVIEW

File: content/tools/polyfill.md [hunk 1]
```diff
@@ -27,16 +27,25 @@ summary: |
 
 {% import 'embed.macros.njk' as embed %}
 
-## Cascade Layers Polyfill
+## Popover Polyfill
```
Comment (jgerigmeyer) [inline content/tools/polyfill.md:30]: @SondraE I think we could consider adding a screenshot from the Popover (and also Layers?) demo pages, like we do for anchor positioning?

Comment (SondraE) [reply content/tools/polyfill.md:30]: Done

**Review by SondraE — COMMENTED** [review]
lgtm

**Comment by jgerigmeyer** [comment]
> @jgerigmeyer want to fix the build here?

@mirisuzanne It's fixed: https://deploy-preview-588--oddleventy.netlify.app/polyfill/

**Pull request merged**